winui/Avatar-view/visual-styles.md

```diff
@@ -1,10 +1,364 @@
 ---
 layout: post
 title: Visual Styles in WinUI AvatarView control | Syncfusion
-description: This section will explain about the different types of visual styles that exists in WinUI AvatarView.
+description: Learn here all about the different visual styles that are available in the WinUI AvatarView.
```

Comment:
Description: Learn about the different visual styles available in the WinUI AvatarView.